plugins/modules/luks_device.py

```diff
@@ -217,6 +217,47 @@
             - "Will only be used on container creation."
         type: int
         version_added: '1.5.0'
+    perf_same_cpu_crypt:
+        description:
+            - "This option allows user to perform encryption using the same cpu that IO was submitted on."
+            - "The default is to use an unbound workqueue so that encryption work is automatically balanced between available CPUs."
+            - "This option is only relevant for open action."
+        type: bool
+        default: false
+        version_added: '2.3.0'
+    perf_submit_from_crypt_cpus:
+        description:
+            - "This option allows user to disable offloading writes to a separate thread after encryption."
+            - "There are some situations where offloading write bios from the encryption threads to a single thread degrades performance sig‐
+              nificantly."
+            - "The default is to offload write bios to the same thread."
+            - "This option is only relevant for open action."
+        type: bool
+        default: false
+        version_added: '2.3.0'
+    perf_no_read_workqueue:
+        description:
+            - "This option allows user to bypass dm-crypt internal workqueue and process read requests synchronously."
+            - "This option is only relevant for open action."
+        type: bool
+        default: false
+        version_added: '2.3.0'
+    perf_no_write_workqueue:
+        description:
+            - "This option allows user to bypass dm-crypt internal workqueue and process write requests synchronously."
+            - "This option is only relevant for open action."
+        type: bool
+        default: false
+        version_added: '2.3.0'
+    persistent:
+        description:
+            - "If used with LUKS2 devices and activation commands like open or refresh,
+               the specified activation flags are persistently written into metadata and
+               used next time automatically even for normal activation."
+            - "No need to use cryptab or other system configuration files."
```

Comment:
Isn't this only used when opening containers? That should probably be mentioned somewhere. Also the options that are affected by this should be mentioned, and the documentations for these options should mention `persistent`.